edit.js

```diff
@@ -16,7 +16,9 @@ var references = require('./services/references'),
   plugins = require('./services/plugins'),
   eventify = require('eventify'),
   link = require('./services/deep-link'),
-  connectionLostMessage = 'Connection Lost. Changes will <strong>NOT</strong> be saved.';
+  connectionLostMessage = 'Connection Lost. Changes will <strong>NOT</strong> be saved.',
+  behaviorReq = require.context('./behaviors', false, /^\.\/[\w\-]+\.js$/), // no *.test.js
+  decoratorReq = require.context('./decorators', false, /^\.\/[\w\-]+\.js$/); // no *.test.js
```

Comment:
use `require.context` to automatically find new behaviors and decorators as we add them, rather than having to manually specify them below (this is a feature webpack gives us that browserify did not)